### PR TITLE
 add Create, Edit, Delete a post

### PR DESCRIPTION
**Pull request by hsnice16**
### This PR will add code for the Create, Edit and Delete a post

In this PR, **243 additions 88 deletions** lines of code
- some files contain only 5 to 6 lines of code change
- some files are just related to backend files

### Please review this PR and give your feedback

---

### CODE REVIEW

**Review by b0llu — APPROVED** [review]
Nothing to suggest. Really well written code. Liked that you are using variables for most values ass well!

File: src/components/broadcast-box/BroadcastBox.jsx [hunk 1]
```diff
@@ -2,15 +2,16 @@ import PropTypes from "prop-types";
 import { Box } from "@mui/material";
 import classNames from "classnames";
 import { BroadcastBoxData } from "data";
-import { useEffect, useState } from "react";
 import { getTimeDurationToShow } from "utils";
 import styles from "./BroadcastBox.module.css";
```
Comment (Dev-AkshatGupta) [inline src/components/broadcast-box/BroadcastBox.jsx:6]: Great that you are using styled components

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything seems quite good to me you can merge it

**Comment by hsnice16** [comment]
Thank you for reviewing,

**Pull request merged**